addon/components/x-select.js

```diff
@@ -77,7 +77,24 @@ export default Ember.Component.extend({
       this._updateValueSingle();
     }
 
-    this.sendAction('action', this.get('value'), this);
+    this.sendAction('action', this.get('value'), this); // should we deprecate `action` in favor of `on-change`?
+    this.sendAction('on-change', this.get('value'), this);
```

Comment:
`action` is still the name of the attribute that holds the default action in the native ember `input` component, and so I think we should hold close to that.